Src/GUI/Plugins/PlotterPlugin.py

```diff
@@ -109,43 +111,54 @@ def build_panel(self):
         sizer0 =  wx.BoxSizer(wx.VERTICAL)
         
         if self.standalone:
-            self.datasetlist = wx.ListCtrl(self.setup, wx.ID_ANY,style = wx.LC_REPORT|wx.LC_SINGLE_SEL)
+            splitterWindow = wx.SplitterWindow(self.setup, style=wx.SP_LIVE_UPDATE)
+            splitterWindow.SetMinimumPaneSize(100)
```

Comment:
I was wondering if a value of 50 would not be more appropriate here. That is just about the height of the header of the table, so it will still prevent the overlapping issue, while at the same time giving users greater freedom with how the two parts of the Data panel are distributed. What do you think?
The current minimum size: 
![image](https://user-images.githubusercontent.com/78977041/162167117-f0ed6c06-82b3-4c37-8132-7d7fc7852b10.png)
The proposed minimum size:
![image](https://user-images.githubusercontent.com/78977041/162167290-57e4414c-5ef4-456f-97c5-4890dc53a83f.png)